spacify-portal/src/main/java/org/uci/spacifyPortal/controllers/UserController.java

```diff
@@ -19,11 +25,78 @@ public class UserController {
     @Autowired
     private OwnerService ownerService;
 
+    @Autowired
+    private CreateRoomService roomService;
+
     @Autowired
     private UserService userService;
 
+    @Autowired
+    private IncentiveService incentiveService;
+
+    @Autowired
+    private SubscriberService subscriberService;
+
     @GetMapping("/eligibleOwners")
     public List<UserEntity> getAllEligibleOwners() {
         return this.userService.getAllEligibleOwners();
     }
+
+    private List<RoomDetail> getOwnedRoomDetails(String userId) {
+        List<OwnerEntity> ownerEntities = this.ownerService.getAllOwnedRooms(userId);
```

Comment:
Instead of creating service files for all tables that we have, we need to autowire the repository and call the the function here.

Reply:
We need to do it for all the Entity-Service files created

Reply:
I did not understand... you want to auto wire repo in controller? Isn't that against Spring design principles?

Reply:
I did not understand... you want to auto wire repo in controller?